src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -381,6 +392,73 @@ private async Task<bool> IsInProcNet8Enabled()
             return isInProcNet8Enabled;
         }
 
+        /// <summary>
+        /// Check local.settings.json to determine what value of FUNCTIONS_WORKER_RUNTIME is.
+        /// </summary>
+        private async Task<string> GetFunctionsWorkerRuntime()
```

Comment:
Don't we have an existing helper method to do this ? `DetermineTargetFramework` ?

Reply:
Replaced with existing helper method `DetermineTargetFramework`